src/components/messenger/lib/types.ts

```diff
@@ -1,11 +1,16 @@
+import { Wallet } from '../../../store/authentication/types';
+
 export interface Item {
   id: string;
   name: string;
   image?: string;
+  primaryZID?: string;
+  wallets?: Wallet[];
```

Comment:
NOTE: In order to display the users public wallet address, some follow up changes will be required in the backend to return the users wallets in the search results.